### ISSUE TITLE
SectionList typescript sections error 

### ISSUE THREAD

**Issue by hengkx** [jemise111/react-native-swipe-list-view#530]
```
SectionListProps<ItemT, SectionT = DefaultSectionT>
```

**Comment by jemise111** [comment]
Hey @hengkx what is the issue? Any chance you could submit a PR to fix? Thanks!

**Comment by tmirun** [comment]
I think @hengkx refer this problem:

when we use `useSectionList`

renderItem the argument should return follow object
https://reactnative.dev/docs/sectionlist#requiredrenderitem
- 'item' (object) - the item object as specified in this section's data key
- 'index' (number) - Item's index within the section.
- 'section' (object) - The full section object as specified in sections.
- 'separators' (object) - An object with the following keys:
  - 'highlight' (function) - () => void
  - 'unhighlight' (function) - () => void
  - 'updateProps' (function) - (select, newProps) => void
    - 'select' (enum) - possible values are 'leading', 'trailing'
    - 'newProps' (object)

https://reactnative.dev/docs/sectionlist#requiredrenderitem

```
<SwipeListView
  useSectionList
  renderItem={({item, section}) => { // section
    return <ItemShowingLayout item={item} />;
  }}
/>
```

**Comment by alexcbluk** [comment]
That's correct as i'm facing the same exact issue as described by @tmirun, while waiting for a fix. Is there a short term fix to convert the data from `ListRenderItemInfo ` to `SectionListRenderItemInfo`. Something like:

```
<SwipeListView
            {...props}
            useSectionList
            renderItem={(data, rowMap): ReactElement => {
                    const newData = { ...data } as SectionListRenderItemInfo<T>  //<---- Something like this?
                    return renderItem({ data, rowMap, updateSectionData, visibleComponent, buttonHeight })
                }
            }
/>
```

**Comment by ab-bee** [comment]
I'm getting the same issue as well.
